usbfreedom/partition.py

```
@@ -0,0 +1,284 @@
+                'dd', 'if=/dev/zero', f'of={self.device_path}',
+                'bs=1M', 'count=1', 'conv=fsync'
+            ], check=False)
+
+            # Get device size
+            result = run_command(['blockdev', '--getsize64', self.device_path])
+            size = int(result.stdout.strip())
+
+            # Last MB
+            run_command([
+                'dd', 'if=/dev/zero', f'of={self.device_path}',
+                f'bs=1M', f'seek={size // (1024*1024) - 1}', 'count=1', 'conv=fsync'
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)